src/common/components/Currency/CurrencyValue.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { KeyValue } from '@folio/stripes/components';
+
+const CurrencyValue = ({ value }) => (
+  <KeyValue
+    label={<FormattedMessage id="ui-invoice.invoice.currency" />}
+    value={value}
```

Comment:
second thing, as I wrote before, https://github.com/folio-org/ui-invoice/pull/347#discussion_r487035723 it looks like not translated value is displayed here

Reply:
Yep. Should be translated. Thx